torch/nn/modules/pooling.py

```diff
@@ -95,7 +95,8 @@ class MaxPool2d(_MaxPoolNd):
     planes.
 
     In the simplest case, the output value of the layer with input size :math:`(N, C, H, W)`,
-    output :math:`(N, C, H_{out}, W_{out})` and :attr:`kernel_size` :math:`(kH, kW)`
+    output :math:`(N, C, H_{out}, W_{out})` or with input size :math:`(C, H, W)`,
```

Comment:
Note that the update here doesn't fit with the math below, as the math references `N_i`. I'd suggest leaving this part as-is.